.travis.yml

```diff
@@ -8,6 +8,8 @@ jobs:
     - stage: test
       language: node_js
       node_js: [node]
+      before_install:
```

Comment:
Travis has a [special way](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions) to configure node versions. If not specified, it defaults to latest stable. Updating manually with `npm install -g npm@latest` does not take benefit of the Travis cache that allow for faster boot.

The existing problem with CI does not come from the test stage but from the release stage, which use `Dockerfile` to build the release.

Reply:
You may remove this, I'll patch the docker image directly.